src/aleph/vm/orchestrator/views/__init__.py

```diff
@@ -57,7 +57,7 @@
 logger = logging.getLogger(__name__)
 
 
-def run_code_from_path(request: web.Request) -> Awaitable[web.Response]:
+async def run_code_from_path(request: web.Request) -> web.Response:
```

Comment:
What is the issue with this ?

Reply:
It doesn't await for the returning response, so needs to await it